pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -262,14 +263,12 @@ func (r *ReconcilePerconaServerMongoDB) Reconcile(request reconcile.Request) (re
 		}
 	}
 
-	for i, replset := range cr.Spec.Replsets {
-		// multiple replica sets is not supported until sharding is
-		// added to the operator
-		if i > 0 {
-			reqLogger.Error(nil, "multiple replica sets is not yet supported, skipping replset %s", replset.Name)
-			continue
-		}
+	repls := cr.Spec.Replsets
+	if cr.Spec.Sharding.Enabled && cr.Spec.Sharding.ConfigsvrReplSet != nil {
+		repls = append(repls, cr.Spec.Sharding.ConfigsvrReplSet)
+	}
 
+	for _, replset := range repls {
```

Comment:
we shouldn't allow replicasets with `replset.Name="cfg"`.
this name is reserved for config server.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/439/commits/d1957dc7e87a294849789c5c53b30495a941c3f4